src/app/components/ProjectsView.tsx

```diff
@@ -0,0 +1,206 @@
+'use client';
+
+import ProjectButton from './ProjectButton';
+
+import { useState, useEffect } from 'react';
+import { TrashIcon } from '@heroicons/react/24/outline';
+import { Dialog } from '@headlessui/react';
+import { usePlanshipCustomer } from '@planship/planship-react';
+
+function load(key: string) {
+  const item = window.sessionStorage.getItem(key);
+  return item != null ? JSON.parse(item) : null;
+}
+
+export default function ProjectsView() {
+  const { entitlements } = usePlanshipCustomer();
+  let [newProjectName, setNewProjectName] = useState('New project');
+  let [newProjectType, setNewProjectType] = useState('Single');
+  let [isOpen, setIsOpen] = useState(false);
+  let [projectsLoaded, setProjectsLoaded] = useState(true);
+
+  const [projects, setProjects] = useState([
+    { name: 'First project', slug: 'first-project', type: 'Single' },
+    {
+      name: 'Second project',
+      slug: 'second-project',
+      type: 'Single',
+    },
+  ]);
+
+  const canCreateProject = () =>
+    entitlements?.['max-projects'] > projects.length;
+
+  useEffect(() => {
+    if (!projectsLoaded) {
+      window.localStorage.setItem('projects', JSON.stringify(projects));
+    }
+    setProjectsLoaded(false);
+  }, [projects, projectsLoaded]);
+
+  useEffect(() => {
+    const projects = window.localStorage.getItem('projects');
+    if (projects) {
+      setProjects(JSON.parse(projects));
+    }
+  }, []);
+
+  const createProject = () => {
+    setProjects([
+      ...projects,
+      {
+        name: newProjectName,
+        slug: newProjectName,
```

Comment:
Should this be `newProjectSlug`?

Reply:
No, this is a local Clicker project list, nothing to do with any Planship stuff. I'm removing this slug prop, unnecessary here (I use it in my Nuxt sample).